source/includes/_one-time-sync-individual-custom-check.html.md.erb

```diff
@@ -0,0 +1,491 @@
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+
+
+### Response fields
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>errors</code></td>
+      <td>
+        An array containing one or more
+        <a href="#check-errors">Errors</a>
+      </td>
+      <td>No</td>
+      <td>
+        A list of errors that occurred while running this check. If this is
+        provided and it''s not empty, the result of this check will be
+        considered to be errored by PassFort, regardless of the other contents
+        of this response.
+      </td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An array containing one or more warnings
+      </td>
+      <td>No</td>
+      <td>
+        Essentially the same as <code>errors</code>, except the
+        <code>sub_type</code> and <code>data</code> fields are not present, and
+        providing warnings will not cause PassFort to consider the check
+        as errored.
+      </td>
+    </tr>
+    <tr>
+      <td><code>provider_data</code></td>
+      <td>
+        Any valid JSON value
+      </td>
+      <td>No</td>
+      <td>
+        This should be the structured JSON data returned by the data provider,
+        or a conversion of the data provider''s response, to allow
+        PassFort to investigate issues that may arise with your integration.
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+>This endpoint should return JSON in the following format:
+
+```json
+<%= pretty_json("responses/run_check_sync_individual_custom", "json") %>
+```
+#### `check_output`
+
+<table>
+  <thead>
+    <th>Field</th>
+    <th>Type</th>
+    <th>Required?</th>
+    <th>Description</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>decision</code></td>
+      <td><code>DecisionClass</code></td>
+      <td>Yes</td>
+      <td>
+        The check will return one of the following decisions: "PASS | FAIL | PARTIAL | WARN | ERROR"
+      </td>
+    </tr>
+    <tr>
+      <td><code>summary</code></td>
+      <td>
+      Some sort of summary of the result
+        in <a href="#data-structure">the PassFort API</a>.
+      </td>
+      <td>If a match was found by the provider</td>
+      <td>A best effort should be made to populate all fields supported by the provider</td>
+    </tr>
+    <tr>
+        <tr>
+      <td><code>external_resources</code></td>
+      <td>
+        An object 
+        in <a href="#data-structure">the PassFort API</a>.
+      </td>
+      <td>No</td>
+      <td>A best effort should be made to populate all fields supported by the provider</td>
+    </tr>
+    <tr>
+      <td><code>warnings</code></td>
+      <td>
+        An object containing the registry data, structured as it is
+        in <a href="#data-structure">the PassFort API</a>.
+      </td>
+      <td>If a match was found by the provider</td>
```

Comment:
"If" to "Whether"

Reply:
"was" to "is"

Reply:
I kept if instead of changing it to whether. I kept if because it establishes a condition - that warning are returned only if a match is returned by the provider. Whether seemed to imply that warnings are required whether or not a match is returned by the provider, which I don't believe is the case. I'm sorry this comment now looks like a grammar lecture; I had to go on Merriam Webster to check whether I was using if correctly 😂